routing-java-ortools/src/main/java/com/nextmv/example/Main.java

```diff
@@ -0,0 +1,110 @@
+package com.nextmv.example;
+
+import com.google.ortools.Loader;
+import com.google.ortools.constraintsolver.Assignment;
+import com.google.ortools.constraintsolver.FirstSolutionStrategy;
+import com.google.ortools.constraintsolver.RoutingIndexManager;
+import com.google.ortools.constraintsolver.RoutingModel;
+import com.google.ortools.constraintsolver.RoutingSearchParameters;
+
+import com.google.ortools.constraintsolver.main;
+import com.google.protobuf.Duration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class Main {
+  public static void main(String[] args) {
+    // record start time of the program
```

Comment:
```suggestion
    // Record start time of the program.
```

Quick comment style alignment.